scheduling-manager-workflows/templates/output.yml

```diff
@@ -0,0 +1,75 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Should this file be checked in?

Reply:
I wasn't sure if we want to check it in so we know what exactly we're deploying? What do you think?

Reply:
Since these will refer to user-specific resources that aren't necessary accessible to others, and probably shouldn't be deployed to production, I think we should generate them every time. Otherwise I'll have a noisy diff each time I run a deployment, even if I didn't change anything.

Reply:
What do you mean they shouldn't be deployed to production? Also, the only time the output template will change if the code changes and generates a new jar. This way you know explicitly that you've made changes to the template and are deploying a new version.

Reply:
However, I'm not dead set on checking this in. Just wondering what you mean by it shouldn't be deployed to production.

Reply:
Sure, my main concern is [this line](https://github.com/blox/blox/pull/254/files/6e163026060b85ace31bfcd6849fa7b58e3cc837#diff-be2bdc987d76ac6407f30b834de53337R23):

```
CodeUri: s3://us-east-1-blox-scheduler-manager/f0c12d0a9d4ed8b32b50c3a2fe104f40
```

That line refers to an S3 bucket owned by the account of whoever last ran `aws cloudformation package`, containing code that was built on a dev laptop. I don't think that is what we actually want to ship to production. Diff-wise, it will also change any time a different bucket is used, or the jar is rebuilt with slightly different timestamps.

Since that's also the only thing that's different from the input template, I'd say that a CR of the input template is enough change control for this.

Reply:
sure I'll remove it